src/CoreBundle/Serializer/SerializerHandlerInterface.php

```diff
@@ -13,22 +13,22 @@
 
 namespace Sonata\CoreBundle\Serializer;
 
-if (!interface_exists(\Sonata\Serializer\SerializerHandlerInterface::class, false)) {
+if (!interface_exists(\Sonata\Form\Serializer\SerializerHandlerInterface::class, false)) {
     @trigger_error(
-        'The '.__NAMESPACE__.'\SerializerHandlerInterface class is deprecated since version 3.13.0 and will be removed in 4.0.'
-        .' Use Sonata\Serializer\SerializerHandlerInterface instead.',
+        'The '.__NAMESPACE__.'\SerializerHandlerInterface class is deprecated since version 3.19.0 and will be removed in 4.0.'
```

Comment:
```suggestion
        'The '.__NAMESPACE__.'\SerializerHandlerInterface class is deprecated since sonata-project/core-bundle 3.19.0 and will be removed in version 4.0.'
```